demo/app/src/components/trade-dialog.tsx

```diff
@@ -20,38 +20,29 @@ import { Input } from "@/components/ui/input";
 import { Form, FormField, FormItem, FormLabel } from "@/components//ui/form";
 import { symbols } from "@/config/trade";
 import { useDatafeed } from "@/app/datafeed-provider";
-import { Pair, chainlinkPairToFeedId } from "@/_types";
+import { Pair } from "@/_types";
 import { useState } from "react";
 import {
   wethConfig,
-  proxyConfig,
-  usdcConfig,
-  avaxConfig,
+  swapAppConfig,
+  usdcConfig, oracleConfig,
 } from "@/config/contracts";
 import { Check } from "lucide-react";
 
 const formSchema = z.object({
   from: z.coerce.number().gt(0),
-  to: z.coerce.number().gt(0),
+  to: z.coerce.number(),
```

Comment:
shell we leave it as it was before?

Reply:
no, as we don't have the price yet , it is blocking trade, will add back when price will be added